packages/catalog-search/src/SearchSuggestionItem.jsx

```diff
@@ -3,43 +3,60 @@ import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 const SearchSuggestionItem = ({
-  title, url, highlightedTitle, authoringOrganization, programType,
-}) => (
-  <Link to={url} key={title} className="suggestion-item">
-    <div>
-      {/* eslint-disable-next-line react/no-danger */}
-      <div dangerouslySetInnerHTML={{ __html: highlightedTitle }} />
-      {
+  title, url, highlightedTitle, authoringOrganization, programType, handleSubmit, disableSuggestionRedirect, hit,
```

Comment:
if the only use of `disableSuggestionRedirect` is to make it call `handleSubmit` then i think we can do with just one prop instea dof two.

e.g., if handleSubmit is renamed to `suggestionItemHandler` or so, you can only call it if it's defined, else do the default behavior

Reply:
like 

```
if (typeof suggestionItemHandler === 'function') { .. your new stuff }
else { old stuff }
```